recipes/py-earth/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: pyearth
+  version: "0.1"
+
+source:
+    url: https://github.com/mehdidc/py-earth/archive/test_conda.zip
+    sha256: f3eca1b2e4ee408255433236df5f4d6c7854c5910214af91824a70eb17e41024
+
+build:
+    number: 0
```

Comment:
Please add the following to `build`.

```
  script: python setup.py install --single-version-externally-managed --record record.txt
```

Then please drop `build.sh` and `bld.bat`.